### PR TITLE
Array signatures fix

### PR DESCRIPTION
**Pull request by HosseinYousefi**
Java signature for `String[]` should be `[Ljava/lang/String;` but it was `L[java/lang/String;`. This PR fixes (🤞) this issue.

### CODE REVIEW

File: jnigen/lib/src/bindings/common.dart [hunk 1]
```diff
@@ -308,19 +308,44 @@ String getInternalName(String binaryName) {
...
+    case "short":
+      return "S";
+    case "boolean":
+      return "Z";
+    default:
+      return 'L${binaryName.replaceAll(".", "/")};';
+  }
+}
+
 String getInternalNameOfUsage(TypeUsage usage,
```
[the remark this answers is not in the document]
Comment (HosseinYousefi) [reply jnigen/lib/src/bindings/common.dart]: Done!

File: jnigen/test/bindings_test.dart [hunk 2]
```diff
@@ -81,6 +81,12 @@ void main() async {
     expect(Example.addInts(10, 15), equals(25));
   });
 
+  // TODO(HosseinYousefi): For now we're passing null as arrays.
```
Comment (mahesh-hegde) [inline jnigen/test/bindings_test.dart:84]: Is this TODO outdated?

**Review by mahesh-hegde — APPROVED** [review]
Looks good to me.

**Review by dcharkes — APPROVED** [review]

**Comment by dcharkes** [comment]
Thanks @HosseinYousefi and @mahesh-hegde ! 🚀

**Pull request merged**